salt/payload.py

```diff
@@ -172,6 +173,33 @@ def dumps(self, msg, use_bin_type=False):
                              Since this changes the wire protocol, this
                              option should not be used outside of IPC.
         '''
+<<<<<<< HEAD
```

Comment:
Looks like a merge wart.

Reply:
Oh yeah, thought I removed both sides of that, whoops. Fixing.

Reply:
Fixed and rebased.